app/assets/javascripts/components/story/StoryItem.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import classname from 'classnames';
+import PropTypes from 'prop-types';
+import {
+  classIconRule,
+  iconRule,
+  isStoryDontEstimated,
+  IsShow,
+  estimateRule,
+  labelSplit,
+  isRelease,
+} from '../../rules/story';
+
+const StoryPoints = () => (
+  <div>
+    <span className="Story__estimate">1</span>
+    <span className="Story__estimate">2</span>
+    <span className="Story__estimate">3</span>
+    <span className="Story__estimate">5</span>
+    <span className="Story__estimate">8</span>
+  </div>
+)
+
+const StateActions = ({ story_type, estimate }) => (
+  <div className='Story__actions'>
+    <IsShow logic={isStoryDontEstimated(story_type, estimate)}>
+      <StoryPoints />
+    </IsShow>
+    <IsShow logic={!isStoryDontEstimated(story_type, estimate)}>
+      <ButtonStart />
+    </IsShow>
+  </div>
+)
+StateActions.propTypes = {
+  story_type : PropTypes.string.isRequired
+}
+
+const ButtonStart = () => (
+  <button type="button" className="Story__btn Story__btn--start">start</button>
+);
+
+const StoryEstimate = ({ estimate }) => (
+  <span className='Story__estimated'>{estimateRule(estimate)}</span>
+);
+
+StoryEstimate.propTypes = {
+  estimate: PropTypes.number,
+};
+
+StoryEstimate.defaultProp = {
+  estimate: '-',
+};
+
+const StoryIcons = ({ story_type }) => (
+  <span className='Story__icon'>
```

Comment:
It's a single icon, right? It would be better to name this component in the singular.